### DISCUSSION TITLE
Node A did not stop normally, resulting in data in the database for picked and picked-by, but Node B was unable to retrieve the task execution. When node A was restarted, the task still could not be re executed

### DISCUSSION THREAD

**Discussion by zhougengx** [kagkarlsson/db-scheduler#562] [Q&A]
Node A did not stop normally, resulting in data in the database for picked and picked-by, but Node B was unable to retrieve the task execution. When node A was restarted, the task still could not be re executed
example

![Uploading image.png…]()

**Comment by kagkarlsson** [comment]
This question needs to be more detailed. Also I suggest reading about dead execution handling in the readme
